packages/react-components/react-teaching-popover-preview/src/components/TeachingPopover/renderTeachingPopover.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { PopoverProvider } from '@fluentui/react-popover';
+import { TeachingPopoverProvider } from '../../TeachingPopoverContext';
+import type { TeachingPopoverState } from './TeachingPopover.types';
+
+/**
+ * Render the final JSX of TeachingPopover
+ */
+export const renderTeachingPopover_unstable = (state: TeachingPopoverState) => {
+  const {
+    appearance,
+    arrowRef,
+    contentRef,
+    inline,
+    mountNode,
+    open,
+    openOnContext,
+    openOnHover,
+    setOpen,
+    size,
+    toggleOpen,
+    trapFocus,
+    triggerRef,
+    withArrow,
+    inertTrapFocus,
+    totalPages,
+    currentPage,
+    setCurrentPage,
+    setTotalPages,
+    onFinish,
+    onPageChange,
+  } = state;
+
+  return (
+    <PopoverProvider
+      value={{
+        appearance,
+        arrowRef,
+        contentRef,
+        inline,
+        mountNode,
+        open,
+        openOnContext,
+        openOnHover,
+        setOpen,
+        toggleOpen,
+        triggerRef,
+        size,
+        trapFocus,
+        inertTrapFocus,
+        withArrow,
```

Comment:
This does not follow Fluent patterns, it should be wrapper in `use*ContextValues()` hook (check `renderMenu()` & `useMenuContextValues_unstable()`).

Reply:
This is an extension of popover that uses the same interface, we will need to modify usePopover to follow this format also.

Reply:
Setup TeachingPopover context to match menu implementation, for now popover will have a slightly different format included in state.